### PR TITLE
Added CENX company as a user

### LINKED ISSUE #50: Add CENX as a user
Officially received approval to add my company to the users section!

### PR DESCRIPTION
**Pull request by bkhouri**
Got approval to add my company, CENX, listed under the robot framework users.

fixes #50

### CODE REVIEW

**Comment by bkhouri** [comment]
fixes #50

**Comment by bkhouri** [comment]
Please don't merge until I confirm the image with my company.

**Comment by bkhouri** [comment]
@Tattoo  @pekkaklarck 
Can we get this PR merged?

**Comment by Tattoo** [comment]
Hi @bkhouri , 

Few comments/questions:

Why is the logo for different thing than the description? The company is CENX, but the logo is "cortx"?

The logo does not follow the sizes of other logos ie. max height 68px, making it stand out on the page.

I'm afraid the description is also too wordy for the layout. Frankly, this bit: 

> with sophisticated real-time and historical analytics, comprehensive visualization, real-time operations, and closed-loop automation capabilities.

... does not really have anything to do with how you are using Robot Framework. Maybe this could be removed from the description text, thus shortening it?

Do you want to implement based on my suggestion, or should I do my version of how the entry in the list could look?

**Comment by rsmparthiban** [comment]
Hi,
 Could you please help me to add my company under the robot framework users.

**Comment by Tattoo** [comment]
@rsmparthiban are you @bkhouri or from CENX?

**Comment by pekkaklarck** [comment]
@rsmparthiban You'd like to add a new organization as a user, right? Please submit a new issue about it.

**Comment by bkhouri** [comment]
@Tattoo I'm from CENX.

I know the image does not show CENX, but that's the image the marketing team wants to use - don't ask me why.  cortx is the product. CENX is the company.

For the text,  used whatever the marketing team wanted.

Feel free to update the PR with your revisions.

**Comment by Tattoo** [comment]
@bkhouri ,

I made the changes detailed in my comment. If you want to work on them more, please open a new issue.

Thanks for contributing!
